M2/Macaulay2/m2/examples.m2

```diff
@@ -39,8 +39,9 @@ capture String := s -> (
     User#"private dictionary" = new Dictionary;
     OutputDictionary = new GlobalDictionary;
     dictionaryPath = {
-	currentPackage.Dictionary,
 	User#"private dictionary",
+	currentPackage.Dictionary,
```

Comment:
I'm suspicious of the idea of setting `currentPackage` while running examples.  Its purpose is to inform the code which package is currently being loaded, so documentation and tests can get stored inside the package.  Thus, if you have an example whose purpose is to demonstrate documentation or tests, that junk will get stored inside the package, instead of inside `User`, where its presence will not be noticed.

Reply:
Where am I setting currentPackage?

Reply:
Hmm, you're right: I thought you were, for some reason.

Reply:
Oh, I can unset it right before calling capture though. That might help remove a few `no-capture-flag`ged nodes. Sorry if you meant this I misunderstood your point.

Reply:
No, I was just confused.

But actually, by the way, isn't CurrentPackage always set to User when examples are being run with "capture"?  So it seems that adding `currentPackage.Dictionary` to the path serves no purpose.

Reply:
It depends. When you run capture from the interpreter, it is set to User, but when you run it while installing a package, it is set to that package, so it does serve a purpose.

Reply:
Ah, okay.